students/AntonBakaiev/hw-8/task-2.js

```diff
@@ -0,0 +1,10 @@
+const createArraysWithObjects = (...names) => {
+    let humansArr = []
```

Comment:
You shouldn’t create additional array for `map` method - [info](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map?retiredLocale=uk).